packages/docs-app/src/examples/core-examples/collapseExamples.tsx

```diff
@@ -0,0 +1,109 @@
+/* !
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ */
+
+import dedent from "dedent";
+import * as React from "react";
+
+import { Button, Classes, Collapse, FormGroup, InputGroup } from "@blueprintjs/core";
+import { CodeExample, type ExampleProps } from "@blueprintjs/docs-theme";
+
+export const CollapseBasicExample: React.FC<ExampleProps> = props => {
+    const [isOpen, setIsOpen] = React.useState(false);
+    const code = dedent`
+        function CollapseExample() {
+            const [isOpen, setIsOpen] = React.useState(false);
+
+            return (
+                <div>
+                    <Button intent="primary" onClick={() => setIsOpen(!isOpen)}>
+                        {isOpen ? "Hide Details" : "Show Details"}
+                    </Button>
+                    <Collapse isOpen={isOpen}>
+                        <p>
+                            This is an example of collapsible content. It could include detailed explanations, extra
+                            information, or supporting text for the primary content on the page. When expanded, the content is
+                            fully visible and takes up space in the document layout. When collapsed, it smoothly slides out of
+                            view.
+                        </p>
+                        <p>
+                            You can include multiple paragraphs, images, or any other elements inside a collapse. This example
+                            demonstrates how a longer content block behaves when toggled.
+                        </p>
```

Comment:
This second paragraph is not in the actual code below

Reply:
Ah, good catch! This was leftover from a previous iteration. Fixed in cf59c0b9737189948044edd96a9442059ec45b48